Week1/create-table.js

```diff
@@ -3,7 +3,8 @@ var connection = mysql.createConnection({
   host     : 'localhost',
   user     : 'hyfuser',
   password : 'hyfpassword',
-  database : 'company'
+  database : 'company',
+  // port : 3307
 });
 
```

Comment:
This file should not be a part of the commit/PR

Reply:
OK. I put it in gitignore folder to avoid pushing it and pushed my work again with fixing other spacing and semi-colons according to the comments.